nacculator_cfg.ini.example

```diff
@@ -1,8 +1,9 @@
 [DEFAULT]
 
 [pycap]
+redcap_server: http://<redcap_server>/api/
 token: Your REDCAP Token
-redcap_server: Your Redcap Server
+tokens: token1,token2,token3,token3
```

Comment:
in order to make less work / confusion for those in the community who already use nacculator, i would  just leave it as "token" with a comment on the previous line about how to fill it out with multiple tokens.